src/components/Title/index.jsx

```diff
@@ -0,0 +1,37 @@
+/*
+ Copyright (C) 2021 Shruti Agarwal (mail2shruti.ag@gmail.com)
+
+ SPDX-License-Identifier: GPL-2.0
+
+ This program is free software; you can redistribute it and/or
+ modify it under the terms of the GNU General Public License
+ version 2 as published by the Free Software Foundation.
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License along
+ with this program; if not, write to the Free Software Foundation, Inc.,
+ 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+*/
+
+import React from "react";
+import PropTypes from "prop-types";
+
+// For the title of all the pages
+import { Helmet } from "react-helmet";
+
+const Title = ({ title }) => {
+  return (
+    <Helmet>
+      <title>{title}</title>
```

Comment:
Adding ` | FOSSology` will be a nice touch to UI. `·` can also be used in place of `|`.

Reply:
Just my opinion :slightly_smiling_face:

Reply:
Updated @GMishx, I believe using `|` will be a better option.